app/src/main/java/org/openobservatory/ooniprobe/model/OONITests.java

```diff
@@ -11,6 +11,7 @@
     public static final String HTTP_HEADER_FIELD_MANIPULATION = "http_header_field_manipulation";
     public static final String TCP_CONNECT = "tcp_connect";
     public static final String WEB_CONNECTIVITY = "web_connectivity";
+    public static final String NDT = "ndt";
     public static final String NDT_TEST = "ndt_test";
```

Comment:
Ah, uhm, now I see this is defined both as `NDT` and as `NDT_TEST`: why?

Reply:
same as above